generic/tests/jumbo.py

```diff
@@ -48,15 +82,6 @@ def run(test, params, env):
     if guest_ip is None:
         test.error("Could not get the guest ip address")
 
-    def get_ovs_ports(ovs):
-        '''
-        get the ovs bridge all Interface list.
-
-        :param ovs: Ovs bridge name
-        '''
-        cmd = "ovs-vsctl list-ports %s" % ovs
-        return set(process.system_output(cmd, shell=True).splitlines())
-
     host_mtu_cmd = "ifconfig %s mtu %s"
     netdst = params.get("netdst", "switch")
     host_bridges = utils_net.Bridge()
```

Comment:
I think use `host_bridges  = utils_net.find_bridge_manager("netdst")` maybe better?
Then you can get the bridge backend but not need to define a new function.